components/Submissions/SubmissionForm.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import { Formik, Form, Field } from 'formik'
+import { useMutation } from '@apollo/client'
+import { ADD_PHOTO, GET_CONTEST_DETAIL } from '../../lib/graphql'
+import { useRef } from 'react'
+import { useEffect } from 'react'
+import { z } from 'zod'
+import { toFormikValidationSchema } from 'zod-formik-adapter'
+import ErrorMessage from '../ErrorMessage'
+
+interface ContestSubmissionInput {
+  contest: number
+  picture: string
+  submission_date?: string
+  votes?: string[]
+}
+
+interface PictureInput {
+  user: string
+  picture_path: string
+  likes?: string[]
+}
+
+export default function SubmissionForm({ contestID, setShowAddPhotoForm }) {
```

Comment:
add props interface

Reply:
done

Reply:
not done, please make sure to apply the changes before saying done

Reply:
I'm so sorry, I must have missed it, properly done now